packages/core/src/components/tabs/tabTitle.tsx

```diff
@@ -54,7 +54,7 @@ export class TabTitle extends AbstractPureComponent2<TabTitleProps> {
                 id={generateTabTitleId(parentId, id)}
                 onClick={disabled ? undefined : this.handleClick}
                 role="tab"
-                tabIndex={disabled ? undefined : 0}
+                tabIndex={disabled ? undefined : selected ? 0 : -1}
```

Comment:
tangentially related to this PR but I think if we set `tabIndex={0}` when disabled here the element may not be visible to screen-readers

not 100% sure on that/not set up right now with a good way to test but it's been a working assumption of mine for awhile that disabled elements should remain in the tab order

if so it would possibly would be more appropriate to add to the wrapping div [here](https://github.com/palantir/blueprint/blob/syu/tabs/packages/core/src/components/tabs/tabs.tsx#L161) but again I'm not sure

Reply:
It looks like this code will never set `tabIndex={0}` when it's disabled, though.

Reply:
yeah, ^ was my understanding as well so i thought this was ok